### PR TITLE
Add a title for parameters pages

### LINKED ISSUE #115: Add a title for parameters pages
On a [parameter page](https://legislation.openfisca.fr/prestations.al_plaf_acc.plafond_pour_accession_a_la_propriete_zone_3.menage_ou_isole_avec_4_enfants): 

- I can't see the path towards the parameter. I need this path to use the parameter in my formula code.
- If there is no description for the parameter, the page title is `Aucune description`.

### PR DESCRIPTION
**Pull request by sandcha**
Fixes #115 

For a parameter:
* Update the page title to show the parameter id instead of its description
* Add the parameter id in the page and split it when it's long (but keep it in one selectable line)

### CODE REVIEW

File: src/assets/style.css [hunk 1]
```diff
@@ -1,12 +1,17 @@
+header {
+  word-wrap:break-word;
```
Comment (MattiSG) [inline src/assets/style.css]: Missing space after colon.

File: src/components/parameter.jsx [hunk 2]
```diff
@@ -19,12 +19,14 @@ const Parameter = React.createClass({
     const {parameter} = this.props
     const description = parameter.description || "Aucune description"
     const isScale = (! parameter.values)
+    const multilineId = parameter.id.replace(/[.]/g, '\u200b.')
```
Comment (MattiSG) [inline src/components/parameter.jsx]: > Escape the dot (`\.`) rather than using a single-entity character set (`[.]`), this is less costly.

Comment (MattiSG) [inline src/components/parameter.jsx]: > If `&shy;` works, prefer it over `\u200b`, which is less readable.

File: src/assets/style.css (hunk 1 above)
Comment (MattiSG) [inline src/assets/style.css]: In which cases is this necessary? I thought we added the `&shy;` exactly to avoid breaking mid-word. 🤔

Comment (sandcha) [inline src/assets/style.css]: Removed.

File: src/components/parameter.jsx (hunk 2 above)
Comment (sandcha) [inline src/components/parameter.jsx]: `&shy;` appears in the string. 😕 `\u200b` kept but comment added.

**Comment by MattiSG** [comment]
capture d ecran 2017-08-18 a 17 22 01
capture d ecran 2017-08-18 a 17 22 11

**Pull request merged**